poppy/wfe.py

```diff
@@ -323,3 +324,32 @@ def get_opd(self, wave, units='meters'):
         if units == 'waves':
             opd /= wave.wavelength.to(u.meter).value
         return opd
+
+
+class StatsticalPSDWFE(WavefrontError):
+    """
+    Statistical PSF WFE class from power law for optical noise.
+
+    Input WFE in radians RMS.
+    """
```

Comment:
Doc string should define what the input parameters are. 

(Note that poppy uses the numpy docstring convention, rather than the ReST docstring convention that hicat-package uses. See other poppy classes for examples)